Dockerfile.deps

```diff
@@ -23,42 +23,37 @@ RUN mkdir -p /srv/explorer \
  && apt-get -yqq install ${ESPLORA_BUILD_DEPS} tor ${CORE_BUILD_DEPS}
 
 
-RUN git clone --quiet --depth 1 --single-branch --branch v0.39.0 https://github.com/nvm-sh/nvm.git /root/.nvm \
+RUN git clone --quiet --depth 1 --single-branch --branch v0.40.1 https://github.com/nvm-sh/nvm.git /root/.nvm \
  && rm -rf /root/.nvm/.git \
  && source /root/.nvm/nvm.sh \
- && nvm install v17.1.0
+ && nvm install v18.20.4
 
-# Build core from sources until PR https://github.com/bitcoin/bitcoin/pull/23387 is merged
```

Comment:
This doesn't seem merged, are we currently not using `savemempoolfees`?
or is it ok to upgrade to version `27.0` without this?

Reply:
It seems it's used (or at least there are reference) in the codebase: https://github.com/Blockstream/esplora/blob/4f5c82fa2a1dd00a6b4b40804c4b44b3532469cb/contrib/nginx-sync.conf.in#L18

Reply:
We needed this "complexity" because we had many esplora nodes come and go, and so needed a quick way to sync the fee-estimates, that's the os.execute in nginx you're seeing here.

I think we no longer need it, we don't use it in the new architecture either.

Reply:
The docker `run.sh` entrypoint is using the `fee_estimates` endpoint:

https://github.com/Blockstream/esplora/blob/4f5c82fa2a1dd00a6b4b40804c4b44b3532469cb/run.sh#L301

If we're removing this, it should be removed there too. Also, it would be nice if the `fee_estimates` removal was separated to its own commit (or PR, even).

> we don't use it in the new architecture either

You mean that `SYNC_{SOURCE,SECRET}` (`run.sh`'s 4th/5th arguments) are no longer set in the new architecture?

If that's the case then the mempool endpoint is no longer used either, should we removed it too?

/cc @greenaddress

Reply:
I wondered about leaving mempool in for others that may find it useful, since it doesn't force us to apply a patch and do a custom build.

Reply:
Updated the PR as requested.